staking_deposit/cli/generate_bls_to_execution_change.py

```diff
@@ -0,0 +1,188 @@
+    prompt=lambda: load_text(['arg_validator_start_index', 'prompt'], func=FUNC_NAME),
+)
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda num: validate_int_range(num, 0, 2**32),
+        lambda: load_text(['arg_validator_index', 'prompt'], func=FUNC_NAME),
+    ),
+    help=lambda: load_text(['arg_validator_index', 'help'], func=FUNC_NAME),
+    param_decls='--validator_index',
+    prompt=lambda: load_text(['arg_validator_index', 'prompt'], func=FUNC_NAME),
+)
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda bls_withdrawal_credentials: validate_bls_withdrawal_credentials(bls_withdrawal_credentials),
+        lambda: load_text(['arg_bls_withdrawal_credentials', 'prompt'], func=FUNC_NAME),
+    ),
+    help=lambda: load_text(['arg_bls_withdrawal_credentials', 'help'], func=FUNC_NAME),
+    param_decls='--bls_withdrawal_credentials',
+    prompt=lambda: load_text(['arg_bls_withdrawal_credentials', 'prompt'], func=FUNC_NAME),
+)
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda address: validate_eth1_withdrawal_address(None, None, address),
+        lambda: load_text(['arg_execution_address', 'prompt'], func=FUNC_NAME),
+    ),
+    help=lambda: load_text(['arg_execution_address', 'help'], func=FUNC_NAME),
+    param_decls='--execution_address',
+    prompt=lambda: load_text(['arg_execution_address', 'prompt'], func=FUNC_NAME),
+)
+@jit_option(
+    # Only for devnet tests
+    default=None,
+    help="[DEVNET ONLY] Set specific GENESIS_FORK_VERSION value",
+    param_decls='--devnet_chain_setting',
+)
+@click.pass_context
+def generate_bls_to_execution_change(
+        ctx: click.Context,
+        bls_to_execution_changes_folder: str,
+        chain: str,
+        mnemonic: str,
+        mnemonic_password: str,
+        validator_start_index: int,
+        validator_index: int,
+        bls_withdrawal_credentials: bytes,
+        execution_address: HexAddress,
+        devnet_chain_setting: str,
+        **kwargs: Any) -> None:
+    # Generate folder
+    bls_to_execution_changes_folder = os.path.join(
+        bls_to_execution_changes_folder,
+        DEFAULT_BLS_TO_EXECUTION_CHANGES_FOLDER_NAME,
+    )
+    if not os.path.exists(bls_to_execution_changes_folder):
+        os.mkdir(bls_to_execution_changes_folder)
+
+    # Get chain setting
+    chain_setting = get_chain_setting(chain)
+
+    if devnet_chain_setting is not None:
+        click.echo('\n%s\n' % '**[Warning] Using devnet chain setting to generate the SignedBLSToExecutionChange.**\t')
+        devnet_chain_setting_dict = json.loads(devnet_chain_setting)
+        chain_setting = get_devnet_chain_setting(
+            network_name=devnet_chain_setting_dict['network_name'],
+            genesis_fork_version=devnet_chain_setting_dict['genesis_fork_version'],
+            genesis_validator_root=devnet_chain_setting_dict['genesis_validator_root'],
+        )
+
+    # TODO: generate multiple?
```

Comment:
yes, please kind sir allow for the bulk bonus feature mentioned here https://notes.ethereum.org/YGWpjmGHQceeq1gijiZuJg